lp-steam/templates/csgo/server.j2

```diff
@@ -42,4 +62,27 @@ sv_minupdaterate "{{ item.value.tickrate | default('128') }}"
 sv_minrate 80000
 sv_maxrate 0
 
-exec rules_{{ item.value.rules | default('esl5on5') }}.cfg
+{% if item.value.backup | default(false) %}
+mp_backup_round_auto 1 
+mp_backup_round_file backup 
+mp_backup_round_file_pattern {{ csgo_backup_round_file_pattern | default('%prefix%_%date%_%time%_%map%_round%round%.txt') }}
+{% endif %}
+
+{% if item.value.overtime | default(false) %}
```

Comment:
`maxrounds` and `startmoney` are set to the defaults here, I'd propose to move this to a private/rulebook config.